### ISSUE TITLE
how to install 

### ISSUE THREAD

**Issue by venu942000** [mcu-debug/rtos-views#15]
iam working with stm32 micro controller and aslo iam using cortex debug but in your readme there is installation is given

please provide me any file for installation guide

**Comment by haneefdm** [comment]
Not sure what you mean. You install it from the VSCode marketplace or right from within VSCode extensions section where you manage all extensions. Like every other extension.

https://marketplace.visualstudio.com/items?itemName=mcu-debug.rtos-views

Also, if you install the latest version of Cortex-Debug, it will auto-install it for you ad this is a dependency.

Can you explain a bit more of what is going on?

**Comment by haneefdm** [comment]
Please close this issue if you are doing okay

**Comment by venu942000** [comment]
iam new to vs code 

iam using stm32f4 and i configure everything  and work fine with cotex debug now i need to trace the rtos memory usage so i use rtos view .

when i start the debug , in xrtos tab the view is like ?????? no data is preview . can you understand my problem

**Comment by venu942000** [comment]
i have installed rtos views directly on vs code

**Comment by haneefdm** [comment]
Nope, I don't understand. You have not said which RTOS -- every RTOS is totally different. We sometimes give you hints to say how to get more info. There are dozens of #defines for a given RTOS you have to enable to see all the info.

How about a screenshot (not your whole screen, just the RTOS panel part).

**Comment by venu942000** [comment]
hi

i will send the screenshot today

**Comment by venu942000** [comment]
this happens when debugging the controller []() 
![IMG-20230127-WA0007](https://user-images.githubusercontent.com/123604998/215008372-c9adf313-7dc8-48c5-9f6d-d6f998074b2e.jpg)

**Comment by haneefdm** [comment]
Did you notice on the bottom instructions on getting the missing fields?

**Comment by haneefdm** [comment]
You disable all that info in your FW.

**Comment by venu942000** [comment]
> You disable all that info in your FW.


is there any sample or example

**Comment by haneefdm** [comment]
Did you scroll down or make the window bigger? Instructions should be right there. Expand the hints. Also, read the FreeRTOS docs and header file for configuration.

<img width="799" alt="image" src="https://user-images.githubusercontent.com/41269583/215041867-01473462-f350-4720-a0d7-027745b0c7c4.png">

If the hint is missing we have a different problem.

**Comment by venu942000** [comment]
sorry there is no hint is there . ok i will check my firmware but i dont see any hint in my window

**Comment by haneefdm** [comment]
Unless you can provide a test case where I can duplicate the issue, not much I can do. Here is what I see when info is missing. Of course you have to click on the Blue text to see the full text.

<img width="1227" alt="image" src="https://user-images.githubusercontent.com/41269583/215155656-cc5c57ba-42ea-444a-a43d-ac7b6a07bea4.png">
